Sprintfinity3902/PauseMenu.cs

```diff
@@ -14,7 +14,7 @@ public class PauseMenu : Interfaces.IDrawable, Interfaces.IUpdateable
         private int count;
         private Game1 Game;
         public Player Link;
-        public Map Map { get; set; }
```

Comment:
Is the map no longer needed for the pause menu? whats the change here?

Reply:
The "Map" here is the Sprint 3 picture of the map that we used to teleport to each room. It is no longer needed for Sprint 4!